packages/formstr-app/src/components/v1/Index.tsx

```diff
@@ -0,0 +1,108 @@
+import { useMemo, useState } from "react";
+import QuestionCard from "./QuestionCard/QuestionCard";
+import { Button, Dropdown, Menu, MenuProps } from "antd";
+import { AnswerTypes, Field } from "@formstr/sdk/dist/interfaces";
+import { MenuFoldOutlined, MenuUnfoldOutlined } from "@ant-design/icons";
+import { AnswerTypeLabels } from "./constants";
+import { makeTag } from "../../utils/utility";
+import { createForm } from "@formstr/sdk";
+
+export interface IQuestion extends Field {
+  tempId: string;
+  inputSettings: {};
+}
+
+const initialQuestion: IQuestion = {
```

Comment:
it would be nice to move this into a util file. Just to increase readability.
packages/formstr-app/src/components/v1/QuestionList/Index.tsx -> main comp
packages/formstr-app/src/components/v1/QuestionList/util.tsx -> related utils